sensu/plugins/check-vyatta-tunnels.py

```diff
@@ -59,11 +59,11 @@ def main():
         switch_on_criticality()
 
     for index, device in enumerate(device_list):
-        if " vti" in str(device):
```

Comment:
HI Carl, 

the ip_vti0 is created by kernel and we can ignore. So we just need to care about other vti interfaces. In addition, only vti interfaces added into vpn tunnel can be monitored by snmp. this is to say the interface won't be retrieved by snmp if they are defined but not bonded into any vpn tunnel.  

In addition, I have to remove the space before "vti". otherwise I won't get normal vti interfaces to further check, as normal vti is like "vti0" and there is no space one the front of device name.